test/demo/basic-html-drawer.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```

Comment:
there was no obvious html drawer demo and I realized that I've maybe actually never used it before, so I made one and tested the new rewritten version here.

Reply:
Good idea!